concent-builder/install-secrets.yml

```diff
@@ -0,0 +1,50 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Create a directory for secrets
+          file:
+            path:  "{{ data_dir }}/concent-secrets/{{ cluster }}/"
+            state: directory
```

Comment:
Can this directory really be created by `shared_user`? Doesn't root have to do this?

Reply:
No, because we use builder user in cloud management playbooks. The question is, do we want to change it?

Reply:
Actually I've seen this later in another of your pull requests and I realized that you rely on the fact that `{{ data_dir }}` already exists  and is owned by `shared_user` - `configure.yml` creates it. So it's fine as is. You could just add a comment that says this.

Reply:
The problem is with cloud management playbooks. They use `builder` user, we must adapt it to the current solution. 
Two solution that I see:
- Change they, to building and deploying in home user directory
- Add installation of concent-deployment repository to builder user and then build and deploy as builder user.

Reply:
Which one do you think will require less changes? I'm still undecided but if one of much easier than the other then we could do it for now.

Reply:
I think that the first one requires less changes.

Reply:
OK. Let's change it to build in home directory. In fact. let's just remove `builder` if it's not used for anything else.